### ISSUE TITLE
WCAG AA Success Criteria 4.1.3 Status Messages

### ISSUE THREAD

**Issue by lyubomir-popov** [canonical-web-and-design/vanilla-framework#3322]
Implement status in components that require it:

- Notifications: role="status"
- Validation messages: role="alert"

The following seem to require modifying labels, and placing links leading to fields that fail validation _under_ the submit button. This would be a pattern change, probably would require UX input too. Flagging for discussion, if we decide to do it, I'll open a separate issue.

Source: 
![image](https://user-images.githubusercontent.com/2741678/94566225-03477f80-0262-11eb-8deb-5f1b7959804b.png)
